app/views/news_article/show.html.erb

```diff
@@ -65,10 +65,10 @@
           title: t("components.share_links.share_this_page") %>
       </div>
 
-      <%= render "components/published_dates", {
-        published: @content_item.published,
-        last_updated: @content_item.updated,
-        history: @content_item.history,
+      <%= render "govuk_publishing_components/components/published_dates", {
+        published: display_date(content_item.initial_publication_date),
```

Comment:
In the example page https://govuk-frontend-app-pr-4674.herokuapp.com/government/news/industrial-strategy-launch-to-hardwire-stability-for-investors the date isn't being rendered properly, I think it needs to be `content_item.first_public_at`

Reply:
(after some discussion, this turns out to be a broader problem with the calling for published_dates)
Hopefully fixed by https://github.com/alphagov/frontend/pull/4709